### ISSUE TITLE
add MPO cable types

### ISSUE THREAD

**Issue by romchi** [netbox-community/netbox#13375]
### NetBox version

v3.5.7

### Feature type

Data model extension

### Proposed functionality

Сan you add cable types such as MPO/MTP (Multiple-fiber Push-On/Pull-off).

The multimode cable does not provide a complete understanding of which type is being used. MPO/MTP is like a separate type of cable, which must be distinguished from ordinary patches at the level of DAC, multimode and singlemode.

Proposed cable types to be added:
- MPO/MTP
- MPO/MTP (breakout)
- MPO/MTP (conversion)
- MPO/MTP (OS2)
- MPO/MTP (OM3)
- MPO/MTP (OM4)
- MPO/MTP (OM5)

### Use case

Will allow a more correct description of the physical connection scheme.

### Database changes

Proposed cable types to be added:
- MPO/MTP
- MPO/MTP (breakout)
- MPO/MTP (conversion)
- MPO/MTP (OS2)
- MPO/MTP (OM3)
- MPO/MTP (OM4)
- MPO/MTP (OM5)

### External dependencies

None

**Comment by DanSheps** [comment]
I agree, to a certain extent, however I would not agree with this as a whole.  Instead, MPO/MTP should be:

MPO Type A (OS2, OM3, OM4, OM5)
MPO Type B (OS2, OM3, OM4, OM5)
MPO Type C (OS2, OM3, OM4, OM5)

Breakout should be modelled with splices, I think, and should not be a cable type in and of itself as our cable model is not setup to handle that type of objects.

**Comment by wohali** [comment]
Surely given #13447 it'd be nice to keep this open until it lands? Bad bot.

**Comment by DanSheps** [comment]
Honestly, I think it is better to model the cable type (OS1/2, OM1/2/3/4/5) as a cable and the MPO type (A/B/C) as the connector type.
